adapters/yandex/yandex.go

```diff
@@ -220,6 +257,29 @@ func modifyBanner(banner openrtb2.Banner) (*openrtb2.Banner, error) {
 	return &banner, nil
 }
 
+func modifyVideo(video openrtb2.Video) (*openrtb2.Video, error) {
+	if video.W == nil || video.H == nil || *video.W == 0 || *video.H == 0 {
+		return nil, &errortypes.BadInput{
+			Message: "Invalid size provided for Video",
+		}
+	}
+
+	if video.MinDuration == 0 {
+		video.MinDuration = videoMinDuration
+	}
+	if video.MaxDuration == 0 {
+		video.MaxDuration = videoMaxDuration
+	}
+	if len(video.MIMEs) == 0 {
+		video.MIMEs = []string{"video/mp4"}
+	}
```

Comment:
This check is not needed. MIMEs are required field in OpenRTB request. 
Refer to [this function](https://github.com/prebid/prebid-server/blob/9a7b75aacb66bf38af4a2fc0e95dd511cfb628bd/ortb/request_validator_video.go#L14)